work/covidvu/pipeline/vuhospitals.py

```diff
@@ -0,0 +1,56 @@
+import urllib
+import json
+import pandas as pd
+from tqdm.auto import tqdm
+from covidvu.pipeline.vujson import _dumpJSON
+
+import os
+from os.path import join
+
+from os.path import join
+from time import sleep
+
+# *** constants ***
+STATE_CODES_PATH = join(os.getcwd(), 'stateCodesUS.csv')
+SITE_DATA               = './site-data'
+OUT_FILE_NAME = 'hospital_bed_count.json'
```

Comment:
Dashes for file names, not underscores (underscores are considered alphanum characters, dashes aren't - important when looking at processing huge numbers of files in the file system or an S3 bucket.

Reply:
Thanks for this! Great tip.